widget-reference/input.md

```diff
@@ -7,6 +7,8 @@ description: >-
 
 # Input
 
+![](/.gitbook/assets/input-widget.gif)
```

Comment:
Few changes needed in the gif
- The property pane is not visible in the gif. Please make sure all properties are shown in the gif.
- The size is too big. Please try to get the size of the file down to 2 mb so it loads faster for a user when viewing documentation

Reply:
Sure, I'll do them.

Reply:
![Screenshot 2020-10-05 at 3 20 15 PM](https://user-images.githubusercontent.com/44445191/95065219-5caa2580-071e-11eb-86a6-c5b68fae409b.png)
When I go inside widgets, these properties are no longer available

Reply:
Ill make a gif where I click from here and go inside ?

Reply:
Yea this is a nice starting point